src/types.ts

```diff
@@ -629,12 +632,18 @@ export interface ManifestOptions {
   edge_side_panel?: {
     preferred_width?: number
   }
+
   /**
    * @see https://github.com/WICG/manifest-incubations/blob/gh-pages/scope_extensions-explainer.md
+   * @see https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/scope_extensions
    * @default []
    */
   scope_extensions: {
     origin: string
+    /**
+      * @default 'origin'
+      */
+    type?: StringLiteralUnion<ScopeExtensionsType> | ScopeExtensionsType
```

Comment:
can you remove extra spaces here, in the default value before the `*` and before `*/`:
```ts
    /**
     * @default 'origin'
     */
    type?: StringLiteralUnion<ScopeExtensionsType> | ScopeExtensionsType
```